API.markdown

```diff
@@ -553,6 +553,32 @@ shifted_samples_array = array.array(sound.array_type, shifted_samples)
 new_sound = sound._spawn(shifted_samples_array)
 ```
 
+Here's how to convert to a numpy float32 array:
+
+```python
+import numpy as np
+from pydub import AudioSegment
+
+sound = AudioSegment.from_file("sound1.wav")
+sound = sound.set_frame_rate(1600)
+channel_sounds = seg.split_to_mono()
+samples = [s.get_array_of_samples() for s in channel_sounds]
+
+fp_arr = np.array(samples).T.astype(np.float32)
+fp_arr /= np.iinfo(samples[0].typecode).max
+```
+
+And how to convert it back to an AudioSegment:
+
+```python
+import io
+import scipy.io.wavfile
+
+wav_io = io.BytesIO()
+scipy.io.wavfile.write(wav_io, 16000, fp_arr)
+sound = pydub.AudioSegment.from_wav(io.BytesIO(wav_io.getvalue()))
```

Comment:
can't you use `wav_io.seek(0)` and pass the original `wav_io` to `AudioSegment.from_wav` ?

Reply:
Yes, verified that works and updated the example.